framework/src/media/audio/audio_manager.c

```diff
@@ -128,12 +127,15 @@ struct audio_device_config_s {
 struct audio_resample_s {
 	bool necessary;
 	uint8_t samprate_types;
-	unsigned int from;
-	unsigned int to;
+	unsigned int user_sample_rate;	// sample rate from a user
 	void *buffer;
-	unsigned int buffer_size;
+	uint32_t buffer_size;
 	float ratio;
 	src_handle_t handle;
+	unsigned int user_channel;	// channel info from a user
+	void *rechanneling_buffer;
+	uint32_t rechanneling_buffer_size;
+	uint8_t format;				// TODO format value from the user, format converting will be added later.
```

Comment:
please align name of channel & format.
Add 'user' as prefix or remove

Reply:
Now format -> user_format.